### PR TITLE
Add SV06 Plus profile

### PR DESCRIPTION
**Pull request by Ashok-Varma**
# Description
- Added SV06 Plus Profile based on the Sovol Published profile.
- Updated SV06 Profile with the latest values
- Couple changes to the sovol base profile, based on latest Sovol Profiles 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Printer definition file(s)
- [ ] Translations

# How Has This Been Tested?
- [x] Profiles were tested with my printers and they work great

**Test Configuration**:
Need Sovol SV06 and SV06 plus

# Checklist:

- [x] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [x] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/contributing.md) 
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have uploaded any files required to test this change

### CODE REVIEW

**Comment by Ashok-Varma** [comment]
cc : @jellespijker @rburema

File: resources/definitions/sovol_sv06_plus.def.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            "default_value": [
+                [-44, 16],
+                [-44, -60],
+                [44, -60],
+                [44, 16]
+            ]
+        },
+        "machine_height": { "default_value": 340 },
+        "machine_max_feedrate_z": { "value": 20 },
```
Comment (jellespijker) [inline resources/definitions/sovol_sv06_plus.def.json:34]: Max feedrate on the Z-axis seems to be `10`

https://github.com/Sovol3d/SV06-PLUS/blob/3569ae18457dd8a7447e3bcdbe7a965e8be760be/MarlinSV06Plus/Marlin/Configuration.h#LL940C22-L940C22

**Comment by eropple** [comment]
@jellespijker These changes don't look outlandish to me, but I don't have context on what `acceleration_print` does versus `machine_acceleration`? Maybe I can help, with that in hand.

**Comment by jellespijker** [comment]
By setting the `acceleration_print` to the `machine_acceleration` in the base profile the acceleration values for SV06 is changed from 500 mm^2/s to 1000 mm^2/s.

Basically the machine_acceleration is intended as the machine limits, while the `acceleration_print` is the acceleration that is used during actual printing. Increasing this might result in quicker prints (positive) but can also introduce artifacts like ringing or missed steps (result layer shifting)

**Comment by eropple** [comment]
Aah, gotcha. I mean, granted mine is now on Klipper with input shaping, but my SV06 is at 3600 mm/s^2. 1000 does not seem outlanding for stock, even with potential resonance from longer rails, etc.

I can't test it, as I don't own a SV06 Plus, but it doesn't seem out-there.

**Comment by jellespijker** [comment]
Thanks for your feedback @eropple 

Will merge this PR. Thank you for your contribution @Ashok-Varma the profile should be available in Cura 5.4

**Review by jellespijker — APPROVED** [review]

**Comment by madhat81** [comment]
> 

When will it be avalible? I need a Mac version of Cura with SV06 Plus profile. Thank you

**Comment by rburema** [comment]
Since we haven't made the 5.4.x branch yet, this will be in 5.4 (otherwise it would have been in the next one). The release for all three OS's is done simultaneously. We've currently planned (but this can change!) the 5.4.0 release in July (hopefully early July), with a 5.4-Beta in (late) June.

**Pull request merged**